src/core/client/admin/test/community/community-rtl.spec.tsx

```diff
@@ -253,21 +253,6 @@ it("change user role", async () => {
   expect(resolvers.Mutation!.updateUserRole!.called).toBe(true);
 });
 
-it("can't change role as a moderator", async () => {
```

Comment:
Instead of just removing this test, what do you think about updating it to test that roles can now be changed as expected when the viewer is an org moderator?

Reply:
A new test has been added! src/core/client/admin/test/community/community-rtl.spec.tsx